client/src/PublicRequest.ts

```diff
@@ -12,6 +12,8 @@ export type SingleKeyResult = {
         keyPath: string,
         address: Uint8Array,
     }>;
+    file: boolean;
+    words: boolean;
```

Comment:
Since this result type is used only for `create` and `import` flows, the _correct_ naming would actually be `usedWordsForImport` and `usedFileForImportOrBackedUpFileAfterWordImport`.
hmmm...

Reply:
My suggestion would be to name them `hasFile` and `hasWords` across the board, because that's what we can objectively say.